contracts/distribution/HAMLINKPool.sol

```diff
@@ -698,12 +712,13 @@ contract HAMLINKPool is LPTokenWrapper, IRewardDistributionRecipient {
         super.withdraw(amount);
         emit Withdrawn(msg.sender, amount);
     }
-
+    ///@dev with withdraw(balOf(msg.sender)) for exit, why use a Withdrawn event on the above functions
```

Comment:
I feel emitting an event on a withdraw is pretty standard, lets easier monitoring / bot creation for how assets are moving throughout / regarding YAM. Is there a specific concern regarding this? Or you wondering why not just place emit in `exit()`?